### ISSUE TITLE
Nan gradient for elu and selu.

### ISSUE THREAD

**Issue by racinmat** [FluxML/Zygote.jl#758]
```julia
julia> using Zygote
julia> gradient(x -> sum(Zygote.selu.(x)), [1_000.0])
([1.0507009873554805],)
julia> gradient(x -> Zygote.selu(x), 1_000.0)
(NaN,)
julia> gradient(x -> sum(Zygote.elu.(x)), [1_000.0])
([NaN],)
julia> gradient(x -> Zygote.elu(x), 1_000.0)
(NaN,)
```
the broadcasted version for Selu works because of https://github.com/FluxML/Zygote.jl/blob/master/src/lib/nnlib.jl#L17
but I assume non-broadcasted version should be added and the elu derivation should also be added, right?

**Comment by DhairyaLGandhi** [comment]
+1 for this

**Comment by racinmat** [comment]
Created PR but it would be great if someone reviewed if it's correct.

**Comment by racinmat** [comment]
Closed by #765
